OpenRA.Mods.Common/Widgets/Logic/Editor/MapEditorLogic.cs

```diff
@@ -86,10 +88,28 @@ public MapEditorLogic(Widget widget, ModData modData, World world, WorldRenderer
 			var copypasteButton = widget.GetOrNull<ButtonWidget>("COPYPASTE_BUTTON");
 			if (copypasteButton != null)
 			{
-				copypasteButton.OnClick = () => editorViewport.SetBrush(new EditorCopyPasteBrush(editorViewport, worldRenderer));
+				copypasteButton.OnClick = () => editorViewport.SetBrush(new EditorCopyPasteBrush(editorViewport, worldRenderer, () => copyFilters));
 				copypasteButton.IsHighlighted = () => editorViewport.CurrentBrush is EditorCopyPasteBrush;
 			}
 
+			var copyFilterDropdown = widget.Get<DropDownButtonWidget>("COPYFILTER_BUTTON");
+			copyFilterDropdown.GetText = () =>
```

Comment:
Can we show a static string  "Copy Filters" here instead? This would make its use clearer, and matches the way we display filters in other places (e.g. the mp server list).

Reply:
Updated.